### ISSUE TITLE
Question: any plans on getting OpenId certified ?

### ISSUE THREAD

**Issue by Badisi** [authts/oidc-client-ts#273]
`oidc-client-js` was certified by OpenId [(#oidc-client-js 1.3)](https://openid.net/developers/certified), so I was wondering if you guys are planning to also do so ?

**Comment by brockallen** [comment]
When I did it originally it was free. Now it costs a non-trivial amount, even for FOSS projects.

**Comment by Badisi** [comment]
Thanks @brockallen ! I didn't know about that.

https://openid.net/certification/fees

> Member Certification Fees
> $700.00 USD per new deployment
> 
> Non-Member Certification Fees
> $3500.00 USD per new deployment

That's rough from a non-profit foundation... 😳

**Comment by pamapa** [comment]
It would be really nice to have this library certified. However i am not so sure that is possible due to the fact that we removed the deprecated `implicit flow`. And we will not bring it back! When i look ate the certification page, this all seems to be from 2015, we have soon 2022!

The price should not be a problem as for open source libraries, they still support `waiver of certification fees` as described here: https://openid.net/certification/open-source-project-certification-policy/

**Comment by raman-nbg** [comment]
I don't think that `implicit flow` will block a certification (or waiving of the certification fees).

Also have a look at the already certified relying parties: https://openid.net/certification/#RPs There are some RPs that did not certify the implicit flow. So this shouldn't be a problem, I guess.

**Comment by raman-nbg** [comment]
A first, smaller step could be to get listed as an "Uncertified OpenID Connect Implementation": https://openid.net/developers/uncertified/

**Comment by chandlerkent** [comment]
+1 on this. My company will only use a certified library, which means we are being directed to use oidc-client-js, which we know is deprecated.

Anything we can do to help get this project certified?

**Comment by pamapa** [comment]
I am open to certificate this library and are willing to help and support. If some body of the community would take the lead and process this, I would really appreciate it!

**Comment by Badisi** [comment]
@pamapa, can I contact them on your behalf ?

**Comment by hugogz** [comment]
Any updates on this topic? I'm also interested in knowing if the OIDC certification is being attempted soon. Thanks

**Comment by Badisi** [comment]
@hugogz, thanks for the ping because I completely forgot about this. I'll have a look at it asap

**Comment by alextse** [comment]
Hi there, just wondering if there is any update on this topic? Thanks.

**Comment by ines-lgd** [comment]
Hello everyone, this project is really useful!
Is the certification process ungoing, @pamapa @Badisi ?

**Comment by ruifang** [comment]
Hi there, is there still a plan for this?
